responsible-ai-collaborative/aiid
# Add description, developer, deployer and harmed parties fields

- resolves #798 

### How to test
1. Open [deploy preview](https://feature-display-incident-fields--clara-aiid-staging.netlify.app/)
2. Submit a new Incident Report
3. You should see 4 new fields: **Description**, **Alleged developer of AI system**, **Alleged deployer of AI system**, and **Alleged harmed or nearly harmed parties**. These fields are **not** mandatory on submission.
4. Submit the new Incident Report. (You'll need to select a different language than English and then select English again due to issue #812 
6. Go to the review list > Ask Clara for login details
7. Review your latest Incident Report. The fields should now be mandatory.
8. Try submitting an Incident Report that doesn't have those fields > the red toast should display stating you are missing information.
9. Update the submission with the missing information (developers, deployers or harmed parties) and update.
10. Try submitting again, you should now be able to submit.
